pegasus/sites.v3/code.org/public/help.md

```diff
@@ -1,5 +1,7 @@
+<!-- code.org/help -->
```

Comment:
I don't think you really need these kinds of comments. I prefer to keep things as clean as possible and only put a comment to explain how code works.